### PR TITLE
Add CMake build

### PR DESCRIPTION
**Pull request by isaachier**
It seems the project has no releases. I was going with 1.0.0 because it hasn't changed in a while.

### CODE REVIEW

**Comment by isaachier** [comment]
Also, sent this pull request upstream: https://github.com/imneme/pcg-c/pull/21.

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,228 @@
+cmake_minimum_required(VERSION 3.1)
+
+set(CMAKE_TOOLCHAIN_FILE "${CMAKE_CURRENT_SOURCE_DIR}/cmake/toolchain.cmake"
+    CACHE FILEPATH
+    "Toolchain to use for building this package and dependencies")
+
+project(pcg VERSION 1.0.0)
```
Comment (ruslo) [inline CMakeLists.txt]: > I was going with 1.0.0 because it hasn't changed in a while

It's better to set `0.0.0` to avoid conflicts with future versions.

**Review by ruslo — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (isaachier) [reply CMakeLists.txt]: Got it.

**Review by ruslo — APPROVED** [review]

**Comment by ruslo** [comment]
Released: https://github.com/hunter-packages/pcg-c/releases/tag/v0.0.0-p0

**Pull request merged**